### PR TITLE
http://superhackerboi.github.io/2020-04-04-Live-CSS-Editor

### PR DESCRIPTION
**Pull request by harlessmark**

### CODE REVIEW

**Comment by harlessmark** [comment]
http://superhackerboi.github.io/2020-04-04-Live-CSS-Editor

File: css-editor/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+
+### Deployment
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+
+### `yarn build` fails to minify
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Comment (vramdhanie) [inline css-editor/README.md:68]: Can drop the boilerplate from the readme provided by create-react-app as it does not add value here.

File: css-editor/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "gh-pages": "^2.2.0",
+    "prismjs": "^1.20.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1",
+    "styled-components": "^5.0.1"
+  },
+  "homepage": "http://superhackerboi.github.io/2020-04-04-Live-CSS-Editor",
```
Comment (vramdhanie) [inline css-editor/package.json:16]: This link seem to be broken. Either you did not deploy with `npm run deploy` or the deploy is broken.

File: css-editor/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (vramdhanie) [inline css-editor/public/index.html:10]: Update description with custom value.

File: css-editor/public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (vramdhanie) [inline css-editor/public/index.html:27]: This title should be specific to your app

File: css-editor/src/App.test.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
```
Comment (vramdhanie) [inline css-editor/src/App.test.js:7]: This test was not updated when you edited the App.js file.

File: css-editor/src/logo.svg [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+        <circle cx="420.9" cy="296.5" r="45.7"/>
+        <path d="M520.5 78.1z"/>
+    </g>
+</svg>
```
Comment (vramdhanie) [inline css-editor/src/logo.svg:7]: Can drop this file since you are no longer using it

File: css-editor/src/App.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    setShowHTML(e.target.value);
+  };
+
+  const CSSChange = (e) => {
+    setShowCSS(e.target.value);
+  };
+
+  return (
+    <>
+      <HTMLDisplay HTML={showHTML} StyledDiv={StyledDiv} />
```
Comment (vramdhanie) [inline css-editor/src/App.js:27]: Without seeing the demo I can't be certain but does this update the CSS for the whole page as specified  in the requirements? Or does it just affect this component?

**Review by vramdhanie — COMMENTED** [review]
Wish I could see the demo site. Check on the deploy and comment here. Code looks good!

**Review by TomGower — COMMENTED** [review]
Easy to read and follow along, so easy I didn't have much to say about specific code you wrote. I don't have much experienced with styled components, though.

**Pull request closed without merging**